files/scripts/swss.sh

```diff
@@ -9,7 +9,7 @@ LOCKFILE="/tmp/swss-syncd-lock$DEV"
 NAMESPACE_PREFIX="asic"
 ETC_SONIC_PATH="/etc/sonic/"
 
-DEPENDENT="radv bgp"
+DEPENDENT="radv"
```

Comment:
Might be good to have a comment explaining namespace specific services are added to this list later.

Reply:
sure, I'll add that in!